linetools/isgm/io.py

```diff
@@ -41,3 +50,67 @@ def abssys_from_json(filename):
 
     # Return
     return abs_sys
+
+
+def read_joebvp(filename, coord, llist=None):
+    """
+    Parameters
+    ----------
+    filename : str
+      joeB VP filename
+    coord : SkyCoord
+    llist : LineList, optional
+
+    Returns
+    -------
+    comps : list
+      list of AbsComponent objects
+    """
+    # init
+    if llist is None:
+        llist = LineList('ISM')
+    comps = []
+    # Read
+    vp_data = Table.read(filename, format='ascii')
+    nflags, nidx = np.unique(vp_data['nflag'], return_index=True)
+
+    # Subset by nflag; Build components
+    for nflag in nflags:
+        mt_lines = np.where(vp_data['nflag'] == nflag)[0]
+        if len(np.unique(vp_data['vel'][mt_lines])) != 1:
+            pdb.set_trace()
```

Comment:
This is immediately tripped by the CASBaH parameter file I tried loading because we will occasionally let the velocity centroid of a line flop around a little in the fit due to the COS wavelength uncertainty.  Another reason the clustering analysis is perhaps useful here.

Reply:
gack.  am still resisting to the clustering.
as a human, we had every intent of grouping
these lines together as a component, right?

Reply:
anyways, have made this an optional check and
defaulted to False